setup.py

```diff
@@ -47,7 +47,7 @@
 install_requires = [
     'Flask>=0.10',
     'celery>=3.1;python_version<"3.7"',
-    'celery>=4.3;python_version=="3.7"',
+    'celery>=4.2;python_version=="3.7"',
```

Comment:
Celery 4.2 will not work on Python 3.7 due to the addition of the ``async`` keyword in Python AFAIR

Reply:
@kprzerwa Actually @lnielsen is correct, sorry for the false info. I think the reason tests are passing is because we're not testing with Redis as a storage backend, which would cause [Python 3.7 to fail](https://github.com/celery/celery/issues/4500#issuecomment-408227597).

Reply:
I discussed also with @ntarocco and we decided to close this PR